decorator.rb

```diff
@@ -0,0 +1,7 @@
+require 'nameable'
+class Decorator < Nameable
+  def initialize
+    super(name)
+    @name = name
+  end
```

Comment:
- [ ] Implement the correct_name method that returns the result of the correct_name method of the @nameable
- [ ] Make sure your decorators in action work after implementing the change.